drivers/hmis/app/graphql/types/forms/form_definition.rb

```diff
@@ -118,6 +119,10 @@ def updated_by
       load_last_user_from_versions(object)
     end
 
+    def supports_save_in_progress
```

Comment:
@gigxz contrary to our discussion, I ended up putting this here on the Form Definition instead of on the Assessment, since the Assessment may not exist yet at the point where we make the determination about whether to show the button. ([relevant frontend code](https://github.com/greenriver/hmis-frontend/pull/982/files#diff-daa760be4e03c0307f7d08234bd1dfdf4f24ad26040b196b043da24237def3ebR72-R76))